src/skate.js

```diff
@@ -19,7 +19,19 @@ import version from './version';
  * @returns {undefined}
  */
 var initDocument = debounce(function () {
-  initElements(document.getElementsByTagName('html'));
+  if (hasDomContentLoadedFired) {
+    initElements(document.getElementsByTagName('html'));
+  } else {
+    document.addEventListener('DOMContentLoaded', function initialiseSkateElementsOnDomLoad() {
+      initElements(document.getElementsByTagName('html'));
+    });
+  }
+});
+
+var hasDomContentLoadedFired = false;
+
+document.addEventListener('DOMContentLoaded', function () {
+  hasDomContentLoadedFired = true;
```

Comment:
I think that would mean elements are initialised regardless of whether `initDocument` is called though; so it'd always initialise the document on DOM ready. That would be unnecessary if no skate skate handlers are called.

It'd make the code simpler though, so maybe it's not worth it. I'd be happy to change it if you think the performance in that case isn't an issue.